packages/thumbnail/README.md

```diff
@@ -26,69 +26,167 @@ import { Thumbnail } from '@spectrum-web-components/thumbnail';
 
 ## Sizes
 
-<sp-tabs selected="m" auto label="Size Attribute Options">
-<sp-tab value="xss">Extra Extra Small</sp-tab>
-<sp-tab-panel value="xxs">
+<sp-tabs selected="500" auto label="Size Attribute Options">
+<sp-tab value="50">50</sp-tab>
+<sp-tab-panel value="50">
 
 ```html
-<sp-thumbnail size="xxs">
+<sp-thumbnail size="50">
     <img src="https://place.dog/100/100" alt="Demo Image" />
 </sp-thumbnail>
 ```
 
 </sp-tab-panel>
-<sp-tab value="xs">Extra Small</sp-tab>
-<sp-tab-panel value="xs">
+<sp-tab value="75">75</sp-tab>
+<sp-tab-panel value="75">
 
 ```html
-<sp-thumbnail size="xs">
+<sp-thumbnail size="75">
     <img src="https://place.dog/100/100" alt="Demo Image" />
 </sp-thumbnail>
 ```
 
 </sp-tab-panel>
-<sp-tab value="s">Small</sp-tab>
-<sp-tab-panel value="s">
+<sp-tab value="100">100</sp-tab>
+<sp-tab-panel value="100">
 
 ```html
-<sp-thumbnail size="s">
+<sp-thumbnail size="100">
     <img src="https://place.dog/100/100" alt="Demo Image" />
 </sp-thumbnail>
 ```
 
 </sp-tab-panel>
-<sp-tab value="m">Medium</sp-tab>
-<sp-tab-panel value="m">
+<sp-tab value="200">200</sp-tab>
+<sp-tab-panel value="200">
 
 ```html
-<sp-thumbnail size="m">
+<sp-thumbnail size="200">
     <img src="https://place.dog/100/100" alt="Demo Image" />
 </sp-thumbnail>
 ```
 
 </sp-tab-panel>
-<sp-tab value="l">Large</sp-tab>
-<sp-tab-panel value="l">
+<sp-tab value="300">300</sp-tab>
+<sp-tab-panel value="300">
 
 ```html
-<sp-thumbnail size="l">
+<sp-thumbnail size="300">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="400">400</sp-tab>
+<sp-tab-panel value="400">
+
+```html
+<sp-thumbnail size="400">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="500">500</sp-tab>
+<sp-tab-panel value="500">
+
+```html
+<sp-thumbnail size="500" style="display: flex;">
+    <div class="image-wrapper">
+        <img
+            class="image"
+            style="max-block-size: 100%;
+                max-inline-size: 100%;
+                position: relative;
+                z-index: 1;"
+            src="https://place.dog/100/100"
+            alt="Demo Image"
+        />
```

Comment:
This should work without needing the inline styling and instead should some from `class="image"`

Reply:
Fixed this by using `converter.classToSlotted('spectrum-Thumbnail-image'),` in `spectrum-config.js` which added 
```
::slotted(*) {
    max-block-size: 100%;
    max-inline-size: 100%;
    position: relative;
    z-index: 1;
}
```